tests/contest/runtimetest/src/main.rs

```diff
@@ -44,7 +44,9 @@ fn main() {
         "io_priority_class_be" => tests::test_io_priority_class(&spec, IoprioClassBe),
         "io_priority_class_idle" => tests::test_io_priority_class(&spec, IoprioClassIdle),
         "devices" => tests::validate_devices(&spec),
+        "root_readonly" => tests::test_validate_root_readonly(),
         "no_pivot" => tests::validate_rootfs(),
+
```

Comment:
Remove this extra blank

Reply:
@sat0ken This is still to be done

Reply:
Sorry, I fixed.
https://github.com/youki-dev/youki/pull/2976/commits/a49ca55f3b66d513820022b21b44b20371188ed7